CSharp/CSharpStyleguide.ruleset

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="C# styleguide ruleset" Description="C# styleguide ruleset" ToolsVersion="15.0">
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA1101" Action="Warning" />
+    <Rule Id="SA1121" Action="Error" />
+    <Rule Id="SA1201" Action="Warning" />
+    <Rule Id="SA1202" Action="Warning" />
+    <Rule Id="SA1206" Action="Warning" />
+    <Rule Id="SA1210" Action="Warning" />
+    <Rule Id="SA1300" Action="Error" />
+    <Rule Id="SA1302" Action="Error" />
+    <Rule Id="SA1303" Action="Error" />
+    <Rule Id="SA1309" Action="None" />
+    <Rule Id="SA1500" Action="Error" />
+    <Rule Id="SA1503" Action="None" />

```

Comment:
I know it's a lot a of work, and I'm not asking to do it right now.. but if we could add comments to this file, it will help a lot in case someone needs to customize it (e.g. treating a specific warning as an error)

Comments like:

```<Rule Id="SA1503" Action="None" /> <!-- Braces must not be ommited -->```

The comment would state what the rule is meant for, and the action could change based on each project needs.